### PR TITLE
db/license

### PR DESCRIPTION
**Pull request by sebastianbochan**
- [x] - CSS files
- [x] - TS modules
- [x] - TS classes
- [x] - plugins
- [x] - packages
- [x] - replace dynamic counter with a static value. Will be replaced in the future.

### CODE REVIEW

**Review by pawellysy — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]

File: css/dashboard/gui.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/**
+ * @license Highcharts
```
Comment (pawelfus) [inline css/dashboard/gui.scss:2]: Highcharts Dashboards*

File: css/dashboard/lightDarkTheme.scss [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/**
+ * @license Highcharts
```
Comment (pawelfus) [inline css/dashboard/lightDarkTheme.scss:2]: Highcharts Dashboards*

File: css/datagrid.scss [hunk 3]
```diff
@@ -1,3 +1,11 @@
+/**
+ * @license Highcharts
```
Comment (pawelfus) [inline css/datagrid.scss:2]: Highcharts Dashboards*
(probably in the future will be changed)

**Review by pawelfus — CHANGES_REQUESTED** [review]
Just a minor things

**Review by pawelfus — APPROVED** [review]
Thanks!

**Pull request merged**